docker/Dockerfile

```diff
@@ -2,11 +2,11 @@ FROM debian:buster-slim as base
 
 ENV PHP_CONF_DATE_TIMEZONE=UTC \
     PHP_CONF_MAX_EXECUTION_TIME=60 \
-    PHP_CONF_MEMORY_LIMIT=512M \
+    PHP_CONF_MEMORY_LIMIT=1G \
     PHP_CONF_OPCACHE_VALIDATE_TIMESTAMP=0 \
     PHP_CONF_MAX_INPUT_VARS=1000 \
-    PHP_CONF_UPLOAD_LIMIT=40M \
-    PHP_CONF_MAX_POST_SIZE=40M
+    PHP_CONF_UPLOAD_LIMIT=100M \
+    PHP_CONF_MAX_POST_SIZE=200M
```

Comment:
We doubled the post max size here without a comment. Not sure that it matters but thought I'd point it out

Reply:
The docker file being updated was part of the upgrade script (it just copies it from pim-community-dev).

Reply:
I understand that most of this file was copied, but I think you may have hand edited line 9? I might be misunderstanding, but I thought the `akeneo-custom-php.ini` file was something custom we added. When we removed it in 992c960ec83e5d2fd4d3b5d9e41ceff0b1cc28a1, we replaced it with env vars, and we doubled the `post_max_size`. I wasn't sure if that was intentional, though it probably doesn't matter very much.

Reply:
Hmm.. good point, I have a vague memory of changing it to 200M cause I thought I remembered it needing to be larger than UPLOAD_LIMIT. But you're right, it probably doesn't matter. I'm fine with leaving it at 200M